juliadoc/jldoctest.py

```diff
@@ -480,14 +480,14 @@ def __run(self, test, out):
                 self.julia.stdin.write('_ans=ans; nothing\n'.encode('utf-8'))
                 # read separator
                 sep = 'fjsdiij3oi123j42'
-                self.julia.stdin.write(('println("' + sep + '")\n').encode('utf-8'))
+                self.julia.stdin.write(('println("\\n' + sep + '")\n').encode('utf-8'))
```

Comment:
Technically we don't need this fix if `dump` gets fixed. I definitely still want to include this fix as it seems unreasonable for doctests to hang forever just because the output of a call doesn't have a trailing newline.